test/ssl_test.cc

```diff
@@ -0,0 +1,90 @@
+#include <chrono>
```

Comment:
Thanks @oschaaf . How much of this can be split into separate PRs? I.e. there is a ton of churn, around Python test infra, SSL objects, coverage, general upgrade? Ideally our Envoy upgrades are lightweight and some of these changes, e.g. test infra, are one-offs. Could you split out some of this easily?

Reply:
Yes, this can be split up into multiple PR's easily; be we may have to temporarily allow a lower coverage threshold at some point. I'll carve off a more fun-sized piece right away into a separate PR.